### ISSUE TITLE
da `*` 

### ISSUE THREAD

**Issue by ksherlock** [ksherlock/qasm#21]
`*` needs to be reset after each da/dw/db/etc argument.

merlin:
```
                     1 
008000: 14 00 12 00  2              da   offset-*,offset-* 
008004: 00 00 00 00  3              ds   16           
008008: 00 00 00 00 00 00 00 00 
008010: 00 00 00 00 
                     4 offset                         
                     5 
```
qasm:
```
                     1 
008000: 14 00 14 00  2              da   offset-*,offset-* 
008004: 00 00 00 00  3              ds   16           
008008: 00 00 00 00 00 00 00 00 
008010: 00 00 00 00 
                     4 offset                         
                     5 
```

**Comment by ksherlock** [comment]
tested merlin32 and qasm (c++ version). neither updates * within the line.  Merlin32 adds a label and uses that instead of * making a fix more difficult.

**Comment by ksherlock** [comment]
orca/m and mpw don't update * within the line, either.

**Comment by ksherlock** [comment]
fixed bdbf62fe36c046fe56097296b477b842df3d486b
